src/main/java/kitchenpos/domain/Menu.java

```diff
@@ -10,6 +10,18 @@ public class Menu {
     private Long menuGroupId;
     private List<MenuProduct> menuProducts;
 
+    public Menu() {
+    }
+
+    public Menu(final Long id, final String name, final BigDecimal price, final Long menuGroupId,
+                final List<MenuProduct> menuProducts) {
```

Comment:
지금처럼 매개변수들의 타입이 서로 거의 다른 경우는 괜찮지만, 기본적으로 매개변수가 너무 많으면 특정 매개변수에 잘못된 데이터가 들어가는 실수가 발생할 수 있다고 생각해요. (ex. `public Member(String name, String email, String nickname) {~}` => `new Member(name, nickname, email)`)

그래서 개인적으로 저는 매개변수가 너무 많다 싶으면 아래처럼 한줄에 하나씩 정렬?해요.

```suggestion
    public Menu(final Long id, 
                final String name,  
                final BigDecimal price, 
                final Long menuGroupId,
                final List<MenuProduct> menuProducts) {
```